samples/dnn/object_detection.py

```diff
@@ -62,31 +64,40 @@
 args = parser.parse_args()
 
 args.model = findFile(args.model)
-args.config = findFile(args.config)
-args.classes = findFile(args.classes)
-
-# If config specified, try to load it as TensorFlow Object Detection API's pipeline.
-config = readTextMessage(args.config)
-if 'model' in config:
-    print('TensorFlow Object Detection API config detected')
-    if 'ssd' in config['model'][0]:
-        print('Preparing text graph representation for SSD model: ' + args.out_tf_graph)
-        createSSDGraph(args.model, args.config, args.out_tf_graph)
-        args.config = args.out_tf_graph
-    elif 'faster_rcnn' in config['model'][0]:
-        print('Preparing text graph representation for Faster-RCNN model: ' + args.out_tf_graph)
-        createFasterRCNNGraph(args.model, args.config, args.out_tf_graph)
-        args.config = args.out_tf_graph
-
-
-# Load names of classes
-classes = None
-if args.classes:
-    with open(args.classes, 'rt') as f:
-        classes = f.read().rstrip('\n').split('\n')
-
-# Load a network
-net = cv.dnn.readNet(cv.samples.findFile(args.model), cv.samples.findFile(args.config), args.framework)
+model_length = len(args.model)
+if args.postprocess != 'yolov8':
+    args.config = findFile(args.config)
+    args.classes = findFile(args.classes)
+
+    # If config specified, try to load it as TensorFlow Object Detection API's pipeline.
+    config = readTextMessage(args.config)
+    if 'model' in config:
+        print('TensorFlow Object Detection API config detected')
+        if 'ssd' in config['model'][0]:
+            print('Preparing text graph representation for SSD model: ' + args.out_tf_graph)
+            createSSDGraph(args.model, args.config, args.out_tf_graph)
+            args.config = args.out_tf_graph
+        elif 'faster_rcnn' in config['model'][0]:
+            print('Preparing text graph representation for Faster-RCNN model: ' + args.out_tf_graph)
+            createFasterRCNNGraph(args.model, args.config, args.out_tf_graph)
+            args.config = args.out_tf_graph
+
+    # Load names of classes
+    classes = None
+    if args.classes:
+        with open(args.classes, 'rt') as f:
+            classes = f.read().rstrip('\n').split('\n')
+
+    # Load a network
+    net = cv.dnn.readNet(cv.samples.findFile(args.model), cv.samples.findFile(args.config), args.framework)
+# onnx_yolov8
+else:
+    classes = None
+    if args.classes:
+        with open(args.classes, 'rt') as f:
+            classes = f.read().rstrip('\n').split('\n')
+    net = cv.dnn.readNetFromONNX(args.model)
+###
```

Comment:
Why we need a separate method `readNetFromONNX`? Can `readNet` cover onnx models?

Reply:
```readNet``` can cover onnx models， but we still needs special handling for yolov8 as it doesn't require ```args.cofig``` and ```args.framework```, we will change code to ```net = cv.dnn.readNet(args.model)```

Reply:
`readNet` can handle empty string for config:
```
readNet("model.onnx", "", "onnx")
```